src/common/devcard.ts

```diff
@@ -76,10 +70,12 @@ export async function getDevCardData(
   userId: string,
   con: Connection,
 ): Promise<DevCardData> {
+  const start = new Date(START_OF_DEVCARD).toISOString();
```

Comment:
This is not what I meant actually. I meant that we should subtract a year from now and use it as the start date.
Because by default when you go to the profile, it shows the information of last year (365 days). So I want it to be aligned with the profile.

Reply:
Ah, I see. Cool. Thanks for the clarification, my bad. Let me work it quickly!